spring-kafka/src/main/java/org/springframework/kafka/aot/KafkaRuntimeHints.java

```diff
@@ -140,34 +117,10 @@ public void registerHints(RuntimeHints hints, @Nullable ClassLoader classLoader)
 
 		Stream.of(
 					AppInfo.class,
-					// standard assignors
-					CooperativeStickyAssignor.class,
-					RangeAssignor.class,
-					RoundRobinAssignor.class,
-					StickyAssignor.class,
 					// standard partitioners
 					org.apache.kafka.clients.producer.internals.DefaultPartitioner.class,
-					RoundRobinPartitioner.class,
 					org.apache.kafka.clients.producer.UniformStickyPartitioner.class,
```

Comment:
Why do we still keep this Kafka client classes here?

Reply:
Missing in the metadata repo.